### PR TITLE
[Ready for Review] Creating solo Zero-to-Binder instructions with screengrabs

### PR DESCRIPTION
**Pull request by alexmorley**
Was originally PR #323 by @sgibson91 

### Summary

Create a set of the Zero-to-Binder instructions for a person working by themselves with screengrabs to explain the steps. Also include an instructor guide on running your own Zero-to-Binder workshop in `instructor-guide_zero-to-binder.md`. These notes can be moved into a README for the workshop when instructor guides for the other sessions have been collated.

### List of changes proposed in this PR (pull-request)

* Add `zero-to-binder-solo.md`
* Add `figures/` folder with screengrabs
* Add instructor guide for running a Zero-to-Binder workshop

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

File: workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+This time, click on "Build Logs" in the big, horizontal, grey bar.
+This will let you watch the progress of your build.
+It's useful when your build fails or something you think _should_ be installed is missing.
+
+<html>
+<img src="figures/create_requirements.png" alt="create_requirements" height="615" width="998">
+<img src="figures/build_logs.png" alt="build_logs" height="562" width="871">
+</html>
+
+**N.B.:** Sometimes Binder's build logs prints things in red font, such as warnings that `pip` is not up-to-date (`pip` is always out of date because it's regularly updated!) or installation messages, especially if you're using R.
```
Comment (LouiseABowler) [inline workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md]: "always out of date" -> "often out of date"?

File: workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+#### Small public files
+
+The simplest approach for small, public data files is to add them directly into your GitHub repository.
+They are then directly encapsulated into the environment and versioned along with your code.
+
+This is ideal for files up to **10MB**.
+
+#### Medium public files
+
+To access medium files **from a few 10s MB up to a few hundred MB**, you can add a file called `postBuild` to your repo.
```
[the remark this answers is not in the document]
Comment (LouiseABowler) [reply workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md]: ... and the next section is actually about postBuild! So the link isn't needed as much as I thought. Having said that, the example linked from the Binder docs does illustrate some other uses of the file, such as removing files that the user doesn't need from the built Binder.

File: workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+1) Create a new repo on GitHub called "my-first-binder".
+   * Don't forget to initialise with a README!
+2) Create a file called `hello.py` via the web interface with `print("Hello from Binder!")` on the first line and commit to master
+
+**N.B.:** GitHub repos **must** be _public_ in order to work with Binder.
+
+<html>
+<img src="figures/create_repo.png" alt="create_repo" height="562" width="671">
+<img src="figures/create_hello.png" alt="create_rhello" height="999" width="1006">
```
Comment (LouiseABowler) [inline workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md]: A lot of the images are slightly squashed; could you preserve the aspect ratio somehow? I'm not sure whether full width (`width="100%"`) or specifying only the width in pixels would be better for this particular document.

File: workshops/boost-research-reproducibility-binder/workshop-presentations/instructor-guide_zero-to-binder.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+### Potential Problems
+
+#### mybinder.org goes down
+Unfortunately, there's not much that can be done about this other than to wait for it to come back online or have a back-up plan for the session.
+
+Sometimes mybinder.org does become overwhelmed with a room full of people all trying to launch binders at the same time. Usually waiting a short period and trying again will solve this problem.
+
+#### Attendees using R
+If you have attendees who bring R code, R packages and RStudio does take a particularly long time for Binder to build. `tidyverse` especially is a very popular and large data package that can take a _very_ long time to load. We recommend that R users modify their code to only install the required packages, and not the whole `tidyverse`. For a more long-term solution, you can point R users to the direction of the following resources:
+* https://github.com/binder-examples/rocker
```
[the remark this answers is not in the document]
Comment (sgibson91) [reply workshops/boost-research-reproducibility-binder/workshop-presentations/instructor-guide_zero-to-binder.md]: 👍

File: workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sgibson91) [reply workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md]: 👍

File: workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md (hunk 3 above)
Comment (sgibson91) [reply workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-solo.md]: So I halved the pixel dimensions of the original images to stop them from being massive, but I don't know much else about html images unfortunately 🤷‍♀️

**Comment by sgibson91** [comment]
Ah damn, just realised this PR has come from @alexmorley 's fork. I'm going to merge it and open a new PR with the changes you suggested @LouiseABowler !

**Pull request merged**